nifi-nar-bundles/nifi-rules-action-handler-bundle/nifi-rules-action-handler-service/src/test/java/org/apache/nifi/rules/handlers/TestAlertHandler.java

```diff
@@ -78,7 +78,7 @@ public void setup() throws InitializationException {
     @Test
     public void testValidService() {
         runner.assertValid(alertHandler);
-        assertThat(alertHandler, instanceOf(AlertHandler.class));
+        MatcherAssert.assertThat(alertHandler, instanceOf(AlertHandler.class));
```

Comment:
Can this be changed to a static import? It looks this is the case in several places.